### ISSUE TITLE
Plugin fails when there are no JS files

### ISSUE THREAD

**Issue by devinus** [ksmithut/metalsmith-uglify#12]
`Error: uglify: Failed to minify : TypeError: Cannot read property 'figure_out_scope' of null`

This will happen when you're setting up a project to be able to handle JS files in the future but there are none yet. Workaround is to make a blank file, but I'd rather not have to do that.

**Comment by ksmithut** [comment]
Hmmm... it should already be handling this. What does your metalsmith directory structure look like (without the empty file)?

**Comment by azappa** [comment]
Me too. I got some of js files into ./src/js/*.js. The part for build and watching edits:
``` javascript
  .use(
    uglify({
      concat: 'static/js/ffa.min.js',
      removeOriginal: true
    })
  )
  .use(
    watch({
      paths: {
        './src/**/*': './build/**/*'
      },
      livereload: true
    })
  )
```

everytime I modify any file into ./src/ folder uglifyjs gave me this error.

**Comment by ksmithut** [comment]
@devinus Are you also using `metalsmith-watch`?

**Comment by devinus** [comment]
@ksmithut I am indeed.

**Comment by azappa** [comment]
I'm using `metalsmith-watch` too (I need it for watching changes of files into my src directory).

**Comment by ksmithut** [comment]
@devinus @azappa This has been published to version 1.2.1. Let me know if you have any issues :)
